python/cugraph/cugraph/dask/sampling/neighborhood_sampling.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_distributed_data
+import cugraph.comms.comms as Comms
+import dask_cudf
+import pylibcugraph.experimental as pylibcugraph
+
+
+def call_nbr_sampling(sID,
+                      data,
+                      src_col_name,
+                      dst_col_name,
+                      num_edges,
+                      do_expensive_check,
+                      start_info_list,
+                      h_fan_out,
+                      with_replacement):
+
+    # Preparation for graph creation
+    handle = Comms.get_handle(sID)
+    handle = pylibcugraph.ResourceHandle(handle.getHandle())
+    graph_properties = pylibcugraph.GraphProperties(
+        is_multigraph=False)
+    srcs = data[0][src_col_name]
+    dsts = data[0][dst_col_name]
+    weights = None
+    if "value" in data[0].columns:
+        weights = data[0]['value']
+
+    mg = pylibcugraph.MGGraph(handle,
+                              graph_properties,
+                              srcs,
+                              dsts,
+                              weights,
+                              False,
+                              num_edges,
+                              do_expensive_check)
+
+    return pylibcugraph.uniform_neighborhood_sampling(handle,
+                                                      mg,
+                                                      start_info_list,
+                                                      h_fan_out,
+                                                      with_replacement,
+                                                      do_expensive_check)
+
+
+def convert_to_cudf(cp_arrays):
+    """
+    Creates a cudf DataFrame from cupy arrays from pylibcugraph wrapper
+    """
+    cupy_sources, cupy_destinations, cupy_labels, cupy_indices = cp_arrays
+    # cupy_sources, cupy_destinations, cupy_labels, cupy_indices, cupy_counts = cp_arrays
+    df = cudf.DataFrame()
+    df["sources"] = cupy_sources
+    df["destinations"] = cupy_destinations
+    df["labels"] = cupy_labels
+    df["indices"] = cupy_indices
+    df["counts"] = cupy_counts
```

Comment:
`cupy_counts`  is not referenced

Reply:
It's been temporarily disabled, until a solution has been found for handing this variable which does not move in device memory